building-for-cross-platform.md

```diff
@@ -1,3 +1,119 @@
 ## Writing cross-platform modules and apps
 * http://shapeshed.com/writing-cross-platform-node/
 * https://gist.github.com/domenic/2790533
+
+### Reading and Writing to the Windows Registry in-process from Node.js
```

Comment:
Let's move this into a page of it's own with the aforementioned title, and then make this a subheading of that.